### ISSUE TITLE
VTK build failure on Fedora 33 relating to Python

### ISSUE THREAD

**Issue by verdurin** [easybuilders/easybuild-easyconfigs#13084]
Problem with Python detection during `configure_step`:
```
-- Could NOT find Python3 (missing: Development.Module) (found suitable version "3.9.5", minimum required is "3.2")
CMake Error at CMake/vtkModule.cmake:4145 (message):
  Could not find the Python3 external dependency.
Call Stack (most recent call first):
  Utilities/Python/CMakeLists.txt:49 (vtk_module_find_package)
  ```

**Comment by verdurin** [comment]
This is using EB develop as of `20210608`.

**Comment by branfosj** [comment]
Is this for `VTK-9.0.1-foss-2020b.eb`? I'm suspicious that https://gitlab.kitware.com/vtk/vtk/-/blob/master/CMake/patches/3.18/FindPython/Support.cmake means VTK searches for Python from highest version down and is detecting your system Python (3.9.5) instead of the EB Python (3.8.6).

**Comment by verdurin** [comment]
Yes, it is for that `VTK`, and that does appear to be what's happening.

**Comment by branfosj** [comment]
In #15172 I had it where my easybuild virutalenv python (3.6) was being detected instead of the EB python (3.9).

It looks like adding `-DPython3_ROOT_DIR=$EBROOTPYTHON` fixes the issue for that version of VTK.
